profiles/models_test.py

```diff
@@ -0,0 +1,18 @@
+"""
+Model tests
+"""
+# pylint: disable=no-self-use
+from django.test import TestCase
+from profiles.factories import ProfileFactory, UserFactory
+
+
+class ProfileTests(TestCase):
+    """tests for the profile model"""
+
+    def student_id_on_save(self):
+        """test that a student id is set on save"""
+        user = UserFactory.create()
+        profile = ProfileFactory.create(user=user)
```

Comment:
@aliceriot You may find this code snippet useful: https://github.com/mitodl/micromasters/blob/master/profiles/permissions_test.py#L27

Since we use Django signals to create a profile (see https://github.com/mitodl/micromasters/blob/master/profiles/signals.py#L17) you will probably need to use this `mute_signals` context manager